torch/quantization/fx/quantization_patterns.py

```diff
@@ -762,3 +762,14 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
         setattr(quantizer.modules[parent_name], name, quantized_standalone_module)
         quantizer.modules[node.target] = quantized_standalone_module
         return quantizer.quantized_graph.node_copy(node, load_arg(quantized=input_quantized_idxs))
+
+class NumericSuiteQuantizeHandler(QuantizeHandler):
```

Comment:
can you define this in _numeric_suite_fx.py

Reply:
Make sense as it is only needed for numeric suite.